metron-platform/metron-hbase-server/pom.xml

```diff
@@ -278,6 +282,7 @@
                                         <exclude>org/slf4j/**</exclude>
                                         <exclude>META-INF/maven/org.slf4j/**</exclude>
                                         <exclude>META-INF/license/LICENSE.slf4j.txt</exclude>
+                                        <exclude>org/apache/hadoop/**</exclude>
```

Comment:
Without these changes to `metron-hbase-server/pom.xml`, the coprocessor prevents the region server from being started.  The following exception is thrown in the HBase server logs.
```
***** ABORTING region server node1,16020,1568142532355: The coprocessor org.apache.metron.hbase.coprocessor.EnrichmentCoprocessor threw java.lang.LinkageError: loader constraint violation: loader (instance of org/apache/hadoop/hbase/util/CoprocessorClassLoader) previously initiated loading for a different type with name "org/apache/hadoop/conf/Configuration" *****
Cause:
java.lang.LinkageError: loader constraint violation: loader (instance of org/apache/hadoop/hbase/util/CoprocessorClassLoader) previously initiated loading for a different type with name "org/apache/hadoop/conf/Configuration"
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
        at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
        at org.apache.hadoop.hbase.util.CoprocessorClassLoader.loadClass(CoprocessorClassLoader.java:317)
        at org.apache.hadoop.hbase.util.CoprocessorClassLoader.loadClass(CoprocessorClassLoader.java:289)
        at org.apache.metron.hbase.coprocessor.EnrichmentCoprocessor.getZookeeperUrl(EnrichmentCoprocessor.java:148)
        at org.apache.metron.hbase.coprocessor.EnrichmentCoprocessor.start(EnrichmentCoprocessor.java:123)
        at org.apache.hadoop.hbase.coprocessor.BaseEnvironment.startup(BaseEnvironment.java:72)
        at org.apache.hadoop.hbase.coprocessor.CoprocessorHost.checkAndLoadInstance(CoprocessorHost.java:263)
        at org.apache.hadoop.hbase.coprocessor.CoprocessorHost.load(CoprocessorHost.java:226)
        at org.apache.hadoop.hbase.coprocessor.CoprocessorHost.load(CoprocessorHost.java:185)
        at org.apache.hadoop.hbase.regionserver.RegionCoprocessorHost.loadTableCoprocessors(RegionCoprocessorHost.java:378)
        at org.apache.hadoop.hbase.regionserver.RegionCoprocessorHost.<init>(RegionCoprocessorHost.java:274)
        at org.apache.hadoop.hbase.regionserver.HRegion.<init>(HRegion.java:810)
        at org.apache.hadoop.hbase.regionserver.HRegion.<init>(HRegion.java:710)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at org.apache.hadoop.hbase.regionserver.HRegion.newHRegion(HRegion.java:6930)
        at org.apache.hadoop.hbase.regionserver.HRegion.openHRegion(HRegion.java:7127)
        at org.apache.hadoop.hbase.regionserver.HRegion.openHRegion(HRegion.java:7100)
        at org.apache.hadoop.hbase.regionserver.HRegion.openHRegion(HRegion.java:7058)
        at org.apache.hadoop.hbase.regionserver.HRegion.openHRegion(HRegion.java:7009)
        at org.apache.hadoop.hbase.regionserver.handler.OpenRegionHandler.openRegion(OpenRegionHandler.java:283)
        at org.apache.hadoop.hbase.regionserver.handler.OpenRegionHandler.process(OpenRegionHandler.java:108)
        at org.apache.hadoop.hbase.executor.EventHandler.run(EventHandler.java:104)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
```

Reply:
This is probably worth a code comment to the effect of "hadoop deps from metron-common are not needed here and cause classpath conflicts with HBase 2.0.2"